imglib.py

```diff
@@ -49,13 +46,11 @@ def get_rotate(orientation):
         return 270
     elif orientation == 8:
         return 90
-    return 0
 
 
-def operations_run1(required_img, saving_path, img_size):
-    image_obj = Image.open(required_img)
-    orientation = get_orientation(required_img)
-    degrees = get_rotate(orientation)
+def saving_modified_img(image_obj, path, img_size):
```

Comment:
Still unacceptable name

<img width="270" alt="unacceptable-lemongrab-e1434739410519" src="https://user-images.githubusercontent.com/290948/68125370-85c8a180-ff22-11e9-8e4f-760b182f9bd6.png">

Reply:
Bro, you just posted cringe